unintended_ml_bias/model_bias_analysis.py

```diff
@@ -659,20 +667,26 @@ def plot_metric_heatmap(bias_metrics_results,
     # well. We should consider using HTML tables instead, using
     # DataFrame.style.applymap for styling the table background color.
     save_inline_png(fig, out, bbox_inches='tight')
-    plt.close()
```

Comment:
I'm not sure if we can remove this - it might be important for use with the nightly eval?

Reply:
Just added it back.  I think this was removed because it prevents the heatmap from being shown in the colab when an out file is specified.  However you are right that it might have unintended results for nightly eval

Reply:
Hm yes. Might be worth investigating whether we really need it eventually.